### ISSUE TITLE
Html5 project exports fails to start in dev and canarly

### ISSUE THREAD

**Issue by SnipG** [Scirra/Construct-3-bugs#1929]
## Problem description

Tried to make HTML5 export examples to report other issue to crbug.com. But links fail to start.

## Attach a .c3p
[exportedExamples.zip](https://github.com/Scirra/Construct-3-bugs/files/2298393/exportedExamples.zip)

## Steps to reproduce

1.Open canarly navigate to: https://snipg.github.io/GPUWithWorker1/  or https://snipg.github.io/GPUWithoutWorker1/
2. Exported projects fail to open, does not open in latest dev or canarly.
3. Next 2 links work in dev but not in canarly. https://snipg.github.io/WithoutWorker2/
https://snipg.github.io/WithWorker2/
4.Try out Link below - does open in 68 and dev, but not canarly.
https://www.scirra.com/labs/bugs/spritestressdom/index.html

## Observed result
Html5 exports fail to work on some chrome versions.

## Expected result
Links should work like in chrome 68.

Tried out links in inkognito mode and restarted pc, nothing helped. Not sure if it just my problem or there is some other issue.

## Affected browsers

* Chrome: yes

## System details

<details><summary>View details</summary>

PASTE HERE

</details>

**Comment by SnipG** [comment]
Just did reinstall to chrome canarly, cant even navigate do construct 3 homepage, without it to trow oh snap errors, and it won't load C3 up anymore or half internet links. Messing around and making with thous examples broke something.

**Comment by AshleyScirra** [comment]
Canary can be very flakey since it changes daily and there hasn't been much testing done on it. Dev can be similar but to a lesser extent. To avoid getting sucked in to working on issues that are only passing quirks of Canary, I think it would be best to only investigate issues that reach the beta channel, or are persistent on dev for a couple of weeks.

FWIW as of Canary 70.0.3528.0 the link in part 4 is working for me, so this already looks like a Canary quirk that was already corrected.

**Comment by AshleyScirra** [comment]
That URL works fine for me. Did you export twice to the same URL with different settings? That might confuse the service worker.

**Comment by SnipG** [comment]
When i open that link in chrome Dev, console picture:
![serviceerror01](https://user-images.githubusercontent.com/28864889/44344029-44241600-a498-11e8-9eb4-e36846523f7f.jpg)

Yes, at first i exported twice, but I exported again once and uploaded before i made previous comment, gives same results, asks for workermains.js. Strange if it works for you, but not for me.

**Comment by SnipG** [comment]
Did new  test: opened c3 113 beta - new project, change runtime to C3(worker disabled by default), added some sprite to change angle every tick, HTML5 export and Tested in chrome Beta - it still requests some worker.js
![noworker](https://user-images.githubusercontent.com/28864889/44398447-86a72a80-a54c-11e8-8cd8-f81ae2e2b0f7.jpg)

**Comment by AshleyScirra** [comment]
Can't reproduce anything wrong here. Try incognito mode to make sure you're not running in to SW cache issues.

**Comment by SnipG** [comment]
Steps: chrome beta - incognite mode - 113 beta, c3 runtime, worker disabled, build html5 export. Removed all files from github, pushed in new. Opened clean Chrome beta - incognite mode - went to link - same error 404, workerMain.js missing. Is there any other way to test it?
I also tried to open export link on new pc, with fresh chrome beta with incognite mode, same error.
Could github itself block it, or my ISP network cached it somehow?
